src/Models/AdminModel.php

```diff
@@ -61,4 +61,25 @@ public function deleteUser($id){
         $query->execute();
         return;
     }
+
+    public function insertLogo() {
+        $logo = $_FILES['logo_upload'];
+        $file = $logo['name'];
+        $file_tmp = $logo['tmp_name'];
+
+        $profile_ext = explode('.',  $file);
+        $filecheck = strtolower(end($profile_ext));
+
+        $file_ext_stored = array('jpeg', 'png', 'jpg');
+
+        if (in_array($filecheck, $file_ext_stored)) {
+            $destinationFile = 'assets/images/' . $file;
```

Comment:
Its storing the values the images folder and not the ' images/logo/ ' folder. Can you just correct the destination path? Rest all looks fine

Reply:
I tired that. But the Image is not getting stored in logo folder. Do you know the reason or solution, please share. I will make the required changes then.

Reply:
@RChaubey16  I've updated the changes. Can you please check?

Reply:
@Libbna Checked and everything works perfectly. Merging the PR.